library/ord-tree.lisp

```diff
@@ -278,7 +280,9 @@
                                  (ins left)))
                       ((EQ) None)
                       ((GT) (map (fn (new-right) (balance clr left pivot new-right))
-                                 (ins right)))))))))
+                                 (ins right)))))
+                   (_
```

Comment:
Same as previous; change to `((DoubleBlackEmpty) (error "Found double-black node outside of removal process"))`, modulo wording on the error message.